package-system/pyside2-qt/build_windows.cmd

```diff
@@ -0,0 +1,70 @@
+@echo off
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM 
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM 
+
+REM Prerequisites:
+REM - Visual Studio 2017 is installed
+REM - 7-zip is installed: https://www.7-zip.org/
+
+REM Download the prebuilt clang package from the Qt server. The current version of PySide2 is not compatible with the latest version of clang package.
+echo Downloading the prebuilt clang package from the Qt server
+call curl https://download.qt.io/development_releases/prebuilt/libclang/libclang-release_80-based-windows-vs2017_64.7z -L -o %TEMP_FOLDER%\libclang-release_80-based-windows-vs2017_64.7z
+"C:\Program Files\7-Zip\7z.exe" x %TEMP_FOLDER%\libclang-release_80-based-windows-vs2017_64.7z -o%TEMP_FOLDER%\libclang-release_80-based-windows-vs2017_64 -y

```

Comment:
Consider building more resiliency around the 7zip path. i.e. turning it into a script parameter with this path as a default value in the event a user installs it somewhere else. Another one would be checking the path and printing a brutally clear error message on absence and exiting.

Reply:
Good point. Just made the update to check the 7-zip path and allow users to change that via an environment variable.